google-ads-shaded/pom.xml

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add a section to README.md explaining how and why users would use the artifact defined by this pom.